src/components/KTableData/KTableData.vue

```diff
@@ -464,13 +461,25 @@ const pageSizeChangeHandler = ({ pageSize: newPageSize }: PageSizeChangeData) =>
   offset.value = null
   pageSize.value = newPageSize
   page.value = 1
+}
```

Comment:
question: does it matter you are no longer calling `emitTablePreferences` in the `pageSizeChangeHandler`?

Reply:
No because I'm calling it in `tablePreferencesUpdateHandler` which is triggered every time `KTableView` emits the `@update:table-preferences` event, which is emitted when page size is updated too.